### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by nelforza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # test
-test
+test2
```
Comment (LinArcX) [inline README.md]: I don't like test2. Please change it to test-two.

**Comment by hoomanist** [comment]
what is philosophy of this repo

**Comment by LinArcX** [comment]
@hoomanist hi.
I've created this repo just for testing features of git and github like sending pull requests, review code and etc..

**Comment by hoomanist** [comment]
> @hoomanist hi.
> I've created this repo just for testing features of git and github like sending pull requests, review code and etc..

funny

**Pull request closed without merging**